MegaApiClient/Serialization/GetNodes.cs

```diff
@@ -37,6 +38,8 @@ public GetNodesResponse(byte[] masterKey)
 
     public Node[] Nodes { get; private set; }
 
+    public Node[] Failed { get; private set; }
```

Comment:
I think `UndecryptedNodes` is a more meaningful name

Reply:
Fixed